sdks/python/hatchet_sdk/runnables/task.py

```diff
@@ -57,8 +70,11 @@ def __init__(
         skip_if: list[Condition | OrGroup] = [],
         cancel_if: list[Condition | OrGroup] = [],
     ) -> None:
-        self.fn = fn
-        self.is_async_function = is_async_fn(fn)
+        self.is_durable = is_durable
+
+        self.fn = _fn
+        self.is_async_function = is_async_fn(self.fn)  # type: ignore
```

Comment:
typing here was a huge PITA so I just added some ignores for now - I can fix this later 🤷